int8 ring_number
float32 delta_x
float32 delta_y
float32 delta_z
float32 abs_x
float32 abs_y
float32 abs_z
float32 norm_x
float32 norm_y
uint64 timestamp
int8 possibility

